### ISSUE TITLE
Checkbox selection via keyboard doesn't work

### ISSUE THREAD

**Issue by computapars** [ExactTarget/fuelux#1113]
Users using keyboard navigation to checkboxes cannot select the checkbox. On focus to the checkbox, the focus is triggered but nothing shows when selected/checked via keyboard action.

**Comment by interactivellama** [comment]
This makes our checkbox fail accessibility. :-( Thanks @computapars! We are planning on sunsetting the current checkbox for a control focused on styling and not JS. We may not fix this. If we don't fix it, we will have something accessible in it's place.

**Comment by cormacmccarthy** [comment]
In the upcoming 3.7.0 release we are removing the JS checkbox and radio buttons entirely and replacing with a CSS only solution, which will actually fix this bug.

**Comment by cormacmccarthy** [comment]
New accessible checkbox and radio buttons are now in master and will go out in the next release.
